mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: "mkdocstrings"
 site_description: "Automatic documentation from sources, for MkDocs."
 site_url: "https://mkdocstrings.github.io/"
-repo_url: "https://github.com/mkdocstrings/mkdocstrings"
+repo_url: "https://github.com/mkdocstrings/mkdocstrings/"
```

Comment:
Thanks. Yes, this is definitely a bug in mkdocs_gen_files; I had too much faith in `urljoin` :D
mkdocs-gen-files should gain these exact lines:
https://github.com/mkdocs/mkdocs/blame/17ff566d2a91703396e954efc521d3869472ba79/mkdocs/structure/pages.py#L108-L109
Ah and it's possible to write tests for that too, but I don't have anything close to it there :(